packages/functional-tests/pages/baseTokenCode.ts

```diff
@@ -38,6 +39,7 @@ export abstract class BaseTokenCodePage extends BaseLayout {
   async fillOutCodeForm(code: string) {
     this.checkPath();
     await this.codeInput.fill(code);
+    await expect(this.codeInput).toHaveValue(code);
```

Comment:
Why do we add this expect **curious? I'm thinking it's because when it fails we have a more informative failure message?

Reply:
I'm not sure this is the right solution, but I was seeing this test occasionally fail where it looks like the submit button is hit before the code input registers. Trace here: https://output.circle-artifacts.com/output/job/f8658cf3-77f6-4fb4-add4-3fd39ce094d2/artifacts/0/playwright-report/index.html#?testId=c70f06ddb6671fac8df3-caad2391e2d15ce8ea32

Reply:
Yikes! okay! maybe we should be adding more of these expects after `.fill()` in future!